ui/package.json

```diff
@@ -33,19 +33,24 @@
     }
   },
   "devDependencies": {
+    "Duration.js": "icholy/Duration.js#golang_compatible",
```

Comment:
Any idea if there any other way to do this? i.e via npm? If not maybe we could ask the author if it's possible to do whatever this branch does via an option? Or possibly publish this branch somehow via npm?

Reply:
Is there a reason you think pointing at GH is bad? Just worried it might disappear? Maybe we should specify a sha so we're not just pulling down whatever's the latest in the branch.

Reply:
Not necessarily bad, just the npm registry is there for a reason, and using a registry sits better with me.

SHA would be better I suppose, but both routes still lose semver benefits (npm 6 has improved this I believe) which is one reason, and yes it could theoretically disappear at anytime. Agree its highly unlikely but there you go - not sure what the process is when someone requests an unpublish from npm for publishes older than 24 hours - but I would imagine its harder to delete nowadays.

Anyway no biggie, just thought I'd ask. I had a quick look in npm just incase there was a version of this branch looks like this is the authors recommended way of doing it.